biohansel/cli.py

```diff
@@ -0,0 +1,281 @@
+import logging
+from typing import Union, Optional, Tuple, List
+
+import attr
+import click
+import pandas as pd
+
+from biohansel.subtype import subtype_contigs_samples, subtype_reads_samples, Subtype
+from biohansel.subtype.const import SUBTYPE_SUMMARY_COLS, JSON_EXT_TMPL
+from biohansel.subtype.metadata import read_metadata_table, merge_metadata_with_summary_results
+from biohansel.subtype.subtype_stats import subtype_counts
+from biohansel.subtype.util import get_scheme_fasta, init_subtyping_params
+from biohansel.utils import does_file_exist, collect_inputs
+from biohansel.utils import init_console_logger
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option()
+@click.option('-v', '--verbose', count=True,
+              help="Logging verbosity (-v for logging warnings; -vvv for logging debug info)")
+def cli(verbose):
+    """Subtype with a biohansel scheme or create a scheme for your organism of interest
+    """
+    lvl = init_console_logger(verbose)
```

Comment:
init_console_logger does not currently return any value for lvl, was it supposed to return something?

Reply:
Fixed in d6d227095a9e3e87f5c4acf099734cc80dc901a3